### PR TITLE
polls/PollReszults: rm overwrite that was braking behaviour and add i…

### PR DESCRIPTION
**Pull request by phillimorland**
…n classname for additional styling when less then 1 answer

- removed the overwrite and reimplemented the default behaviour for slick slider with additional styling in mB pull request

### CODE REVIEW

**Comment by phillimorland** [comment]
will also fix spinner

**Review by sabinammm — APPROVED** [review]
Wow - looks really nice now!

File: adhocracy4/polls/assets/PollResults.jsx [hunk 1]
```diff
@@ -156,11 +134,9 @@ export default class PollResult extends React.Component {
                               <div className="poll-slider__answer">
                                 {this.isUserAnswer(slide) && <i className="fas fa-check-circle" />} {slide.answer}
                               </div>
-                              <div className="poll-slider__count">
+                              <div className={this.props.question.other_choice_answers.length > 1 ? 'poll-slider__count--spaced' : 'poll-slider__count'}>
```
Comment (fuzzylogic2000) [inline adhocracy4/polls/assets/PollResults.jsx:137]: Ah, I needed a while to get that. But this only styles the number of the answer, right? So the `3/127`. And not showing the numbers when there is only one thing in the slider comes from the slider and that's the overwrite you removed? Good catch!

**Review by fuzzylogic2000 — APPROVED** [review]
Nice! The code looks good to me and if @sabinammm says the page also looks good, then I will merge!

File: adhocracy4/polls/assets/PollResults.jsx (hunk 1 above)
Comment (phillimorland) [reply adhocracy4/polls/assets/PollResults.jsx:137]: this actually just moves the number to the corner when the slider automatically removes the arrows when there's only 1 slide

**Pull request merged**